app/Models/UserSetting.php

```diff
@@ -44,7 +44,7 @@ class UserSetting extends Model
 
     protected $guarded = [];
 
-    public function user(){
+    public function user() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->